.vs/VSWorkspaceState.json

```diff
@@ -0,0 +1,7 @@
+{
+  "ExpandedNodes": [
+    ""
+  ],
+  "SelectedNode": "\\Dimensions.sln",
+  "PreviewInSolutionExplorer": false
+}
```

Comment:
What is this file?

Reply:
It makes it so the solution can be opened in preview before opening it in the solution. This file can be removed, but was pushed alongside the commit.